<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thomas Vogt</author>
 <class>Arinc665Qt::MediaSetViewerWindow</class>
 <widget class="QMainWindow" name="Arinc665Qt::MediaSetViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
    <normaloff>:/fa/solid/eye.svg</normaloff>:/fa/solid/eye.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="Arinc665Qt::Media::MediaSetViewWidget" name="mediaSetView" native="true"/>
      <widget class="QTableView" name="filePathMapping">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDecompileMediaSet"/>
   <addaction name="actionOpenMediaSetXml"/>
   <addaction name="actionSaveMediaSetXml"/>
  </widget>
  <action name="actionOpenMediaSetXml">
   <property name="icon">
    <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
     <normaloff>:/fa/solid/file-code.svg</normaloff>:/fa/solid/file-code.svg</iconset>
   </property>
   <property name="text">
    <string>Open Media Set XML</string>
   </property>
   <property name="toolTip">
    <string>Opens an ARINC 665 Media Set XML</string>
   </property>
  </action>
  <action name="actionSaveMediaSetXml">
   <property name="icon">
    <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
     <normaloff>:/fa/solid/file-arrow-down.svg</normaloff>:/fa/solid/file-arrow-down.svg</iconset>
   </property>
   <property name="text">
    <string>Save Media Set XML</string>
   </property>
   <property name="toolTip">
    <string>Save ARINC 665 Media Set XML</string>
   </property>
  </action>
  <action name="actionDecompileMediaSet">
   <property name="icon">
    <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
     <normaloff>:/fa/solid/arrow-right-to-bracket.svg</normaloff>:/fa/solid/arrow-right-to-bracket.svg</iconset>
   </property>
   <property name="text">
    <string>Decompile Media Set</string>
   </property>
   <property name="toolTip">
    <string>Decompile ARINC 665 Media Set</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Arinc665Qt::Media::MediaSetViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">arinc665_qt/media/MediaSetViewWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/arinc665_qt_resources.qrc"/>
  <include location="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc"/>
 </resources>
 <connections/>
</ui>
